api/spec/requests/v1/expense_sheets_controller_spec.rb

```diff
@@ -12,21 +12,32 @@
       let(:request) { get v1_expense_sheets_path }
 
       context 'when user is admin' do
-        let(:user) { create :user, :admin }
-        let(:expense_sheets) { create_list :expense_sheet, 3 }
+        let!(:user) { create :user, :admin }
+        let(:expense_sheets) { create_list :expense_sheet, 3, user: user }
```

Comment:
Add parentheses for long create call
```suggestion
        let(:expense_sheets) { create_list(:expense_sheet, 3, user: user) }
```